benchmarks/internal/choose-worker.js

```diff
@@ -1,13 +1,17 @@
 const Benchmark = require('benchmark')
+const { LIST_FORMATTER } = require('./benchmark-utils')
 
 const suite = new Benchmark.Suite()
 
-const LIST_FORMATTER = new Intl.ListFormat('en-US', {
-  style: 'long',
-  type: 'conjunction'
-})
+function generateWorkersArray (numberOfWorkers) {
+  const workers = []
+  for (let i = 0; i < numberOfWorkers; i++) {
+    workers.push(i)
```

Comment:
**nitpick (non-blocking):** Isn't this the same as `[...Array(numberOfWorkers).keys()]`?

Reply:
If that works with plain JS, you can push that change once tested ;)

Reply:
Okay, currently running one benchmark-run.
Seems that it takes a while :watch: